tests/conftest.py

```diff
@@ -175,13 +175,24 @@ def _add_framework(request, app, slug, **kwargs):
 
     def teardown():
         with app.app_context():
+            FrameworkLot.query.filter(FrameworkLot.framework_id == framework.id).delete()
             Framework.query.filter(Framework.slug == slug).delete()
             db.session.commit()
 
     request.addfinalizer(teardown)
     return framework.serialize()
 
 
+def _add_lot_for_framework(request, app, framework_slug, lot_slug):
```

Comment:
In terms of adding lots for other frameworks... I think probably yes because it's desirable to have consistency in what to expect if you use a framework fixture - i.e. as a developer I shouldn't need to know which frameworks come with lots and which don't (and it makes sense that all frameworks should have lots).  I'd be tempted to have adding lots as part of the base "give me a framework" code, rather than as an optional extra.